io/plugins/eu.esdihumboldt.hale.io.gml/src/eu/esdihumboldt/hale/io/gml/writer/internal/GmlWriterUtil.java

```diff
@@ -129,6 +129,35 @@ public static boolean isID(TypeDefinition type) {
 		}
 	}
 
+	/**
+	 * Determines if the given type is an INSPIRE type
+	 * 
+	 * @param ProdDef the type definition
+	 * @return if the type is an INSPIRE type
+	 */
+	public static boolean isInspireType(PropertyDefinition propDef) {
```

Comment:
We want to determine if the feature type of the instance, where the identifier is defined is an INSPIRE type.

I did not yet get to the place where this method is called, but as a `PropertyDefinition` is passed here, it is basically ruled out that your actually pass the instance type here.
The parameter should be of type `TypeDefinition`.

On the type you should also check it's super types, if any uses the INSPIRE namespace, as it could be a type from an INSPIRE extension.